standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaHook.java

```diff
@@ -52,6 +52,7 @@ public interface HiveMetaHook {
   String PROPERTIES_SEPARATOR = "'";
   String MIGRATE_HIVE_TO_ICEBERG = "migrate_hive_to_iceberg";
   String INITIALIZE_ROLLBACK_MIGRATION = "initialize_rollback_migration";
+  String SKIP_METASTORE_ALTER = "skip_metastore_alter";
```

Comment:
Can we have short description for what would happen if this property is placed into the environment context of HMS calls?